js/models/Profile.js

```diff
@@ -7,10 +7,10 @@ export default class extends BaseModel {
   defaults() {
     return {
       about: '',
-      avatarHash: '',
+      avatarHashes: { tiny: '', small: '', medium: '', large: '', original: '' },
```

Comment:
hmm... in general, for nested objects or arrays of objects, I've been using nested models and collections. It offers a few benefits, for example, a nice concise place to put validations, being able to have more granular change events (e.g. app.profile.get('avatarHash).listenTo('change:medium'), etc.

I guess this is a simple enough structure that we may be able to get away with it being a flat object... but for anything more complex, I think we'd want to take advantage of the full API of a nested Model.

(for the listing, I will probably use a nested model. If I discover some more advantages to that approach, I may retrofit this guy)